### ISSUE TITLE
front facing barcode detection 

### ISSUE THREAD

**Issue by davidsvilem** [googlesamples/android-vision#36]
Is it possible to scan barcodes using the front facing camera ?

**Comment by pm0733464** [comment]
Yes.  If you are using the CameraSource class, you'd just need to set it to use the front facing camera:

https://developers.google.com/android/reference/com/google/android/gms/vision/CameraSource.Builder.html#setFacing(int)

**Comment by davidsvilem** [comment]
Thanks, i noticed that scanning with front facing camera not working on all devices. For example it worked on Nexus 5 but no on Samsung Galaxy S5. 
Do you any idea way? 

Thanks

**Comment by pm0733464** [comment]
I haven't seen that before.

Note that the CameraSource class is now open source, if you'd like to dig into this further:

https://github.com/googlesamples/android-vision/blob/master/visionSamples/barcode-reader/app/src/main/java/com/google/android/gms/samples/vision/barcodereader/ui/camera/CameraSource.java
